Workbooks/Images/Logos/FPAdvLogo.svg

```diff
@@ -0,0 +1 @@
80 206.896 C 150.480 207.100,149.935 207.888,149.269 208.646 C 143.028 215.754,131.960 210.363,131.909 200.192 C 131.858 189.785,142.505 184.367,149.199 191.393 L 151.182 193.474 156.829 193.474 L 162.476 193.474 162.233 192.226 C 160.442 183.018,149.620 176.522,138.728 178.119 M183.624 178.135 C 160.534 182.232,159.973 216.486,182.917 221.303 C 193.073 223.436,204.761 217.644,206.737 209.501 L 206.993 208.445 200.876 208.445 L 194.760 208.445 193.829 209.457 C 189.415 214.256,180.232 212.114,177.554 205.662 L 176.957 204.223 192.758 204.223 L 208.559 204.223 208.325 199.904 C 207.519 185.096,196.940 175.772,183.624 178.135 M232.246 178.275 C 229.424 178.910,227.552 179.755,225.294 181.412 L 223.417 182.791 223.417 181.031 L 223.417 179.271 218.042 179.271 L 212.668 179.271 212.668 207.294 L 212.668 235.317 218.234 235.317 L 223.800 235.317 223.800 226.070 L 223.800 216.822 225.003 217.835 C 233.661 225.119,248.096 221.331,253.503 210.354 C 261.554 194.012,248.674 174.576,232.246 178.275 M277.351 178.300 C 259.941 181.510,253.653 203.562,266.674 215.748 C 276.989 225.401,294.786 222.387,300.908 209.951 C 309.185 193.140,295.631 174.930,277.351 178.300 M342.575 178.109 C 340.250 178.602,338.603 179.377,336.564 180.940 L 334.741 182.337 334.741 180.804 L 334.741 179.271 329.559 179.271 L 324.376 179.271 324.376 200.000 L 324.376 220.729 329.942 220.729 L 335.509 220.729 335.509 207.354 C 335.509 194.097,335.517 193.964,336.401 192.287 C 339.249 186.886,347.043 186.795,349.712 192.131 L 350.672 194.050 350.798 207.390 L 350.924 220.729 356.289 220.729 L 361.653 220.729 361.534 205.662 C 361.400 188.668,361.405 188.696,358.173 184.183 C 354.841 179.529,348.155 176.926,342.575 178.109 M307.486 200.000 L 307.486 220.729 313.052 220.729 L 318.618 220.729 318.618 200.000 L 318.618 179.271 313.052 179.271 L 307.486 179.271 307.486 200.000 M191.768 188.627 C 193.837 189.591,196.251 192.077,196.886 193.901 L 197.273 195.010 187.312 195.010 C 181.834 195.010,177.351 194.916,177.351 194.802 C 177.351 190.005,186.402 186.129,191.768 188.627 M74.021 189.279 C 81.484 193.008,82.745 202.747,76.505 208.450 C 68.989 215.320,57.404 209.410,58.060 199.040 C 58.587 190.701,66.807 185.674,74.021 189.279 M238.627 189.186 C 248.905 194.489,245.668 211.116,234.357 211.116 C 224.449 211.116,220.306 197.290,228.191 190.540 C 231.058 188.086,235.402 187.523,238.627 189.186 M286.499 189.250 C 297.149 194.628,293.435 211.359,281.601 211.314 C 270.968 211.273,266.279 197.910,274.392 190.769 C 277.614 187.932,282.641 187.302,286.499 189.250 M13.820 210.173 L 13.820 220.729 19.770 220.729 L 25.720 220.729 25.720 210.173 L 25.720 199.616 19.770 199.616 L 13.820 199.616 13.820 210.173 " stroke="none" fill="#1c242c" fill-rule="evenodd"></path><path id="path2" d="" stroke="none" fill="#108ca0" fill-rule="evenodd"></path><path id="path3" d="" stroke="none" fill="#104468" fill-rule="evenodd"></path><path id="path4" d="" stroke="none" fill="#10482c" fill-rule="evenodd"></path></g></svg>
```

Comment:
Hey @RaGerges, Do not use illegal attributes like data-name, style, **xmlns:link**, please remove  xmlns:link as it is an illegal attribute, thanks.

Reply:
Dear @v-sabiraj . xmlns:xlink removed. Thanks